data/ggjGameStory.xml

```diff
@@ -1,30 +1,213 @@
-         <functionalDescription>
-            <tag id="wealthTheme"/>
-	         </functionalDescription>
-	         <prerequisite>
-	            <quantReq id="wealthMetric" operator="greaterThan" compareTo="1"/>         
-	         </prerequisite>
-	         <choice>
-	            <text>Grant him the funds!</text>
-	            <outcome>
-	               <text>The blacksmith hefts the purse of coins and hurries off, promising great results.</text>
-	               <quantModifier id="wealthMetric" absolute="false">+1</quantModifier>
-	               <tagModifier id="paidBlacksmith">add</tagModifier>
-	            </outcome>
-	         </choice>
-	         <choice>
-	            <text>Decline the offer.</text>
-	            <outcome>
-	               <text>The blacksmith looks dissappointed, but bows slowly and walks away.</text>
-	            </outcome>
-	         </choice>
-	      </storyNode>
-
+			<teaserText>The town blacksmith comes to you with an interesting proposal...</teaserText>
+			<eventText> The blacksmith stands before you, twisting his beard in excitement. He's hoping that you can fund the creation of ornate metal-worked holy symbols for your burgeoning 	faith. </eventText>
+			<functionalDescription>
+				<tag id="wealthTheme"/>
+			</functionalDescription>
+			<prerequisite>
+				<quantReq id="wealthMetric" operator="greaterThan" compareTo="1"/>         
+			</prerequisite>
+			<choice>
+				<text>Grant him the funds!</text>
+				<outcome>
+					<text>The blacksmith hefts the purse of coins and hurries off, promising great results.</text>
+			       		<quantModifier id="wealthMetric" absolute="false">+1</quantModifier>
+			       		<tagModifier id="paidBlacksmith">add</tagModifier>
+				</outcome>
+			</choice>
+			<choice>
+				<text>Decline the offer.</text>
+				<outcome>
+					<text>The blacksmith looks dissappointed, but bows slowly and walks away.</text>
+			    	</outcome>
+			</choice>
+		</storyNode>
+		<storyNode id="crowOmen" type="satellite" lastNode="false">
+			<teaserText>One of the villagers wants to inform you of a strange happening...</teaserText>
+			<eventText>Ulrich said the black crow was lucky, I saw one last night and now my crops are dead.</eventText>
+			<functionalDescription>
+			</functionalDescription>
+			<choice>
+				<outcome>
+					<text>The villagers are a bit more weary of rumors passed on from others, they'd rather defer to you on such matters.</text>
+					<quantModifier id="trustMetric" absolute="false">+1</quantModifier>
+					<quantModifier id="superstitionMetric" absolute="false">-1</quantModifier>
+				</outcome>
+			</choice>
+		</storyNode>
+		<storyNode id="shepFlockBirth" type="satellite" lastNode="false">
+			<teaserText>A happy occurence is cause for consecration...</teaserText>
+			<eventText>Shepard has had a new lamb born and wants you to come over the consecrate the flock.</eventText>
+			<functionalDescription>
```

Comment:
You'll want to add prominence values for any metrics that are involved in the nodes here. The values can just be 1.

Reply:
(This is true for all story nodes. So, here, you'd add the three that appear in the possible outcomes.)